README.md

```diff
@@ -72,24 +72,24 @@ background and may take a significant amount of time to complete.
 
 ## Monitoring data upload progress
 
-For each volume snapshot that is uploaded to S3, an uploads.veleroplugin.io customer resource is generated.  These records contain the current state of an upload request.  You can list out current requests with
+For each volume snapshot that is uploaded to S3, an uploads.cnsdp.vmware.com customer resource is generated.  These records contain the current state of an upload request.  You can list out current requests with
```

Comment:
Should this be uploads.datamover.cnsdp.vmware.com?

Reply:
Done